data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -268,4 +279,400 @@
             {NEW_GOLD_CARRYOVER 40}
         [/endlevel]
     [/event]
+
+# This place was virtually empty, but there were notes indicating something more was intended.
+#
+# Sub-quest: help the wose with the undead (per original notes).
+# Requirements: the undead and wose leaders must still be alive
+# Start Trigger: first meeting with any wose
+# Success Trigger: the death of the undead leader at the hands of side 1; the wose leader must still be alive
+# Complication: other sides might kill the undead leader first.
+# Result: a book appears at (1,1)
+# Effect: taking the book grants movement cost 1 through forest, forest defense 70%, and ambush (like Elvish Ranger)
+# Only some units can read the book. Those who cannot: Delfador, Elves, Dwarves, Merfolk, Horsemen and Mages.
+#
+# I chose these effects because we're close to the end, the next scenario (S22) has some large forests but,
+# thereafter, there is not a lot of use for forest skills, since most of theterritory is open grasslands.
+# So these effects should not have too large an effect upon the remaining parts of the campaign.
+#
+# GL-2016JUL
+
+#po Haralamdum is ancient, and his mind does not work well any more. He takes things very slowly and carefully, even for a Wose. He halts and pauses, often searching for something in his memory, using meaningless syllables to fill the gap. The syllables of his name come from the phrase he uses to mean "that is all" or "I am done" which he says often, usually out of frustration with how hasty everyone seems. Nobody alive knows his real name, and he has likely forgotten as well. Feel free to add, move, or remove pauses to break up, or even scatter, his train of thought. For inspiration, think of Asperger Syndrome.
```

Comment:
Probably I should just leave it out or make it a general comment. I don't really understand PO comments, but thought it might be good, here, to record my thoughts, however they appear.